build/TestShared/xunit.runner.json

```diff
@@ -1,5 +1,5 @@
-{
-  "parallelizeTestCollections": true,
-  "parallelizeAssembly": true,
+﻿{
+  "maxParallelThreads": 1,
+  "parallelizeTestCollections": false,
```

Comment:
revert this, this changes all test projects. I'm guessing this was debugging, if not we can talk about what needs to be done to bring parallel back